reliability/core-it/src/androidTest/kotlin/com/datadog/android/core/integration/tests/InternalSdkCoreContextTest.kt

```diff
@@ -134,7 +132,7 @@ class InternalSdkCoreContextTest : MockServerTest() {
         assertThat(context.appBuildId).isEqualTo(BUILD_ID)
         assertThat(context.source).isEqualTo(ANDROID_SOURCE)
         assertThat(context.processInfo.isMainProcess).isTrue()
-        assertThat(context.networkInfo.connectivity).isEqualTo(NetworkInfo.Connectivity.NETWORK_WIFI)
+        assertThat(context.networkInfo.connectivity).isNotNull()
```

Comment:
why this change is needed? isn't emulator on WiFi by default?

Reply:
you would be surprised but sometimes, especially on lower APIs it returns 3G there :).